test/models/escrowCreate.ts

```diff
@@ -0,0 +1,177 @@
+
+        assert.throws(
+            () => verifyEscrowCreate(invalidAmount),
+            ValidationError,
+            "EscrowCreate: invalid Amount"
+        )
+    })
+
+    it (`invalid CancelAfter`, () => {
+        const invalidCancelAfter = {
+            Account: "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            TransactionType: "EscrowCreate",
+            Amount: "10000",
+            Destination: "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+            CancelAfter: "533257958",
+            FinishAfter: 533171558,
+            Condition: "A0258020E3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855810100",
+            DestinationTag: 23480,
+            SourceTag: 11747
+        } as any
+        
+        assert.throws(
+            () => verifyEscrowCreate(invalidCancelAfter),
+            ValidationError,
+            "EscrowCreate: invalid CancelAfter"
+        )    
+    })
+
+    it (`invalid FinishAfter`, () => {
+        const invalidFinishAfter = {
+            Account: "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            TransactionType: "EscrowCreate",
+            Amount: "10000",
+            Destination: "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+            CancelAfter: 533257958,
+            FinishAfter: "533171558",
+            Condition: "A0258020E3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855810100",
+            DestinationTag: 23480,
+            SourceTag: 11747
+        } as any
+        
+        assert.throws(
+            () => verifyEscrowCreate(invalidFinishAfter),
+            ValidationError,
+            "EscrowCreate: invalid FinishAfter"
+        )    
+    })
+
+    it (`invalid Condition`, () => {
+        const invalidCondition = {
+            Account: "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            TransactionType: "EscrowCreate",
+            Amount: "10000",
+            Destination: "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+            CancelAfter: 533257958,
+            FinishAfter: 533171558,
+            Condition: 0x1411224324,
+            DestinationTag: 23480,
+            SourceTag: 11747
+        } as any
+        
+        assert.throws(
+            () => verifyEscrowCreate(invalidCondition),
+            ValidationError,
+            "EscrowCreate: invalid Condition"
+        )    
+    })
+
+    it (`invalid DestinationTag`, () => {
+        const invalidDestTag = {
+            Account: "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            TransactionType: "EscrowCreate",
+            Amount: "10000",
+            Destination: "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+            CancelAfter: 533257958,
+            FinishAfter: 533171558,
+            Condition: "1411224324",
+            DestinationTag: "23480",
+            SourceTag: 11747
+        } as any
+        
+        assert.throws(
+            () => verifyEscrowCreate(invalidDestTag),
+            ValidationError,
+            "EscrowCreate: invalid DestinationTag"
+        )    
+    })
+
```

Comment:
It'd be good to add a test for each of the for the two cases above:
1. Both CancelAfter and FinishAfter being missing (Should fail validation)
2. Both FinishAfter and Condition being missing (Should fail validation)